build/configs/qemu/README.md

```diff
@@ -35,6 +35,14 @@ qemu-system-arm -M lm3s6965evb -kernel ../build/output/bin/tinyara -nographic
 make download
 ```
 
+## How to debug
+
+```
+arm-none-eabi-gdb
+(gdb) file ../build/output/bin/tinyara
```

Comment:
This means gdb command window.
I used this to distinguish bash command and gdb command